test/Context.spec.js

```diff
@@ -819,5 +819,19 @@ describe('Context', () => {
 
 		refs2.dispatchEvent(new Event('click'))
 		assert.html(container, `<button style="color: orange;">2</button><button>Colors!</button>`)
+	}),
+
+	it('should write and read propTypes of context providers properly', () => {
+		const 
+			TestCtx1 = createContext(),
+			TestCtx2 = createContext(),
+			propTypes1 = {},
+			propTypes2 = {}
+
+		TestCtx1.Provider.propTypes = propTypes1
```

Comment:
consider using [Object.getOwnPropertyDescriptor()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyDescriptor) instead

Reply:
@Zolmeister Mmmh, not really sure whether we really should use ```getPropertyDescriptor``` in the unit tests for this: The fact that the "propTypes" attribute is handled by getter and setter functions internally is just an implementation detail and therefore the aspect how the "propTypes" property is actually handled internally is not part of the public API (may change in future). As the unit tests do only test for the public API we should not use ```getOwnPropertyDescriptor``` IMHO....

Reply:
I see, this is testing `set` and `get` methods, not attribute writeability. disregard